### DISCUSSION TITLE
Release v1.7

### DISCUSSION THREAD

**Discussion by tig** [tui-cs/Terminal.Gui#1904] [General]
I'm starting the process of doing a new release.

The new release will incorporate everything currently in `develop`; the significant change list is as follows:

      Release v1.7.0
      * Moved Terminal.Gui (and NStack) to the github.com/gui-cs organization.
      * Adds multi-step Wizard View for setup experiences (#124)
      * The synchronization context method Send is now blocking (#1854).
      * Fixes #1893: Fix URLs to match gui-cs Org
      * Fixes #1883. Child TopLevels now get Loaded/Ready events.
      * Fixes #1867, #1866, #1796. TextView enhancements for ReadOnly and WordWrap.
      * Fixes #1861. Border: Title property is preferable to Text.
      * Fixes #1855. Window and Frame content view without the margin frame.
      * Fixes #1848. Mouse clicks in Windows Terminal.
      * Fixes #1846. TabView now clips to the draw bounds.
      * Fix TableView multi selections extending to -1 indexes
      * Fixes #1837. Setting Unix clipboard freezes.
      * Fixes #1839. Process WindowsDriver click event if location is the same after pressed and released.
      * Fixes #1830. If "libcoreclr.so" is not present then "libncursesw.so" will be used.
      * Fixes #1816. MessageBox: Hides underlying dialog when visible
      * Fixes #1815. Now returns false if WSL clipboard isn't supported.
      * Fixes #1825. Parent MenuItem stay focused if child MenuItem is empty.
      * Fixes #1812, #1797, #1791. AutoSize fixes.
      * Fixes #1818. Adds Title change events to Window.
      * Fixes #1810. Dialog: Closing event is not fired when ESC is pressed to close dialog.
      * Fixes #1793. ScrollBarView is hiding if the host fit the available space.
      * Added Pos/Dim Function feature to automate layout.
      * Fixes #1786. Windows Terminal is reporting well on mouse button pressed + mouse movement.
      * Fixes #1777 - Dialog button justification. Adds unit tests.
      * Fixes #1739. Setting menu UseKeysUpDownAsKeysLeftRight as false by default.
      * Fixes #1772. Avoids WindowsDriver flickering when resizing.
      * Fixed TableView always showing selected cell(s) even when not focused
      * Fixes #1769. Supports a minimum view size for non-automatic size views.
      * Exposes APIs to support upcoming Web console feature
      * Fixes some scrolling performance issues
      * Fixes #1763. Allowing read console inputs before idle handlers.
      * TableView unicode scenario usability
      * Added unicode testing code to TableEditor

The following Issues need to be resolved before we do this release:

https://github.com/gui-cs/Terminal.Gui/milestone/2

**Comment by BDisp** [comment]
Well, I think that the issues #1861, #1820 and #1816 are fixed for now. The others I let to the more expert than me :-)

**Comment by tig** [comment]
v1.7.0-rc.1 is published

https://www.nuget.org/packages/Terminal.Gui/1.7.0-rc.1

Can someone please give it a try and verify it works? @BDisp or @tznind ?

Thanks.

**Comment by tig** [comment]
v1.7.0 is out the door!

https://github.com/gui-cs/Terminal.Gui/releases/tag/v1.7.0

<img width="321" alt="image" src="https://user-images.githubusercontent.com/585482/181613091-46456deb-320b-4674-b824-24b8b6d1fd45.png">

**Comment by BDisp** [comment]
I'm seeing that the mouse continuous button pressed is not working on `ScrollView`.

**Reply by tig** [reply]
> I'm seeing that the mouse continuous button pressed is not working on `ScrollView`.

Let's get an issue filed!

**Reply by BDisp** [reply]
Done #1921.
